src/component/header/index.tsx

```diff
@@ -43,26 +42,23 @@ function Header(props: HeaderProps) {
     };
   }, [profileDropdown, setProfileDropdown]);
 
-  const navigate = useNavigate();
+  function toggleMobileSidebar() {
+    const nextValue =
+      isVisible === "mobileSidebar" ? undefined : "mobileSidebar";
+    setIsVisible && setIsVisible(nextValue);
+  }
 
   return (
     <PlasmicHeader
       {...rest}
       profileDropdown={profileDropdown}
-      toggleMobileSidebar={{ onClick: () => setIsVisible(mobileSidebar) }}
+      toggleMobileSidebar={{
+        visibility: setIsVisible ? "visible" : "hidden",
```

Comment:
Don't show the sidebar if `setIsVisible` isn't defined as it will cause an error on click otherwise.

I don't like using visibility for this, but it works for now. I need to learn more about the overriding capabilities with plasmic.